mscore/data/shortcuts.xml

```diff
@@ -982,6 +982,14 @@
     <key>toggle-insert-mode</key>
     <seq>Ctrl+I</seq>
     </SC>
+  <SC>
```

Comment:
All eight lines in shortcuts.xml. They will be automatically created when the user assigns a shortcut.